### PR TITLE
Feature/oui datagrid (uu-626)

### PR DESCRIPTION
**Pull request by whispyy**
Changes :
- Tasks
- account-aliases
- disclaimer

### CODE REVIEW

File: src/emailpro/account/alias/emailpro-account-alias.controller.js [hunk 1]
```diff
@@ -5,20 +5,28 @@ angular.module("Module.emailpro.controllers").controller("EmailProTabAliasCtrl",
     "use strict";
```
Comment (JDownloader) [inline src/emailpro/account/alias/emailpro-account-alias.controller.js]: I would personally remove those awful comments

File: src/emailpro/disclaimer/emailpro-disclaimer.controller.js [hunk 2]
```diff
@@ -24,23 +24,31 @@ angular.module("Module.emailpro.controllers")
...
+                            totalCount: disclaimers.count
+                        }
+                    };
+                }).catch((data) => {
                     $scope.setMessage($scope.tr("exchange_tab_DISCLAIMER_error_message"), data.data);
                 });
         };
 
+        $scope.updateDisclaimer = function (disclaimer) {
+            $scope.setAction("emailpro/disclaimer/update/emailpro-disclaimer-update", angular.copy(disclaimer));
```
Comment (JDownloader) [inline src/emailpro/disclaimer/emailpro-disclaimer.controller.js]: I can't believe i'm saying that but, maybe lodash cloneDeep would be a bit more in accordance to our standard.

File: src/emailpro/task/emailpro-task.html [hunk 3]
```diff
@@ -1,68 +1,35 @@
...
+            </div>
+            <div data-ng-if="$row.status == stateCancelled">
+                <span class="fa fa-ban text-muted" aria-hidden="true"></span>
+                <span data-ng-bind-html="tr('exchange_tab_TASKS_'+ $row.status)"></span>
+            </div>
+            <span data-ng-if="$row.status == stateDone"
+                  data-ng-bind-html="tr('exchange_tab_TASKS_'+ $row.status)"></span>
+        </oui-column>
+        <oui-column data-title="tr('exchange_tab_TASKS_table_start_date')" data-property="todoDate" data-sortable="desc">
+            <span data-ng-bind="::$row.todoDate|date:'short'"></span>
```
Comment (JDownloader) [inline src/emailpro/task/emailpro-task.html]: spacing ```"::$row.todoDate | date:'short'">```

File: src/emailpro/task/emailpro-task.html [hunk 4]
```diff
@@ -1,68 +1,35 @@
...
-         data-pagination-server-side-function="loadPaginated"
-         data-pagination-server-side-paginated-stuff="tasksList"
-         data-pagination-server-side-table-loading="tableLoading">
-    </div>
+    <oui-datagrid data-rows-loader="loadPaginated($config)">
+        <oui-column data-title="tr('exchange_tab_TASKS_table_function')" data-property="function">
+            <span data-ng-bind-html="tr('exchange_tab_TASKS_'+ $row.function)"></span>
+        </oui-column>
+        <oui-column data-title="tr('exchange_tab_TASKS_table_state')" data-property="status">
+            <div data-ng-if="$row.status == stateDoing || $row.status == stateTodo">
```
Comment (FredericEspiau) [inline src/emailpro/task/emailpro-task.html]: Type conversion is required here ? (`==` instead of `===`) ?

**Pull request merged**